public/scss/components/CartDetails.module.scss

```
@@ -131,41 +130,175 @@
   }
 }
 
+.itemPrice
+{
+  @include grid-start-(3,2);
+  align-self: center;
+}
+
 .itemSalePrice
 {
   @include SmallSemiBold;
-  color: $color_error
+  color: $color_error;
+}
+
+.itemQty
+{
+  @include grid-start-(2, 2);
 }
 
 .qtyBox
 {
-  @include SmallSmall;
-  display: flex;
-  align-items: center;
+  position: relative;
 
-  input
+  /* Chrome, Safari, Edge, Opera */
+  input::-webkit-outer-spin-button,
+  input::-webkit-inner-spin-button {
+    -webkit-appearance: none;
+    margin: 0;
+  }
+
+  /* Firefox */
+  input[type=number] {
+    -moz-appearance: textfield;
+  }
+
+  input[type="number"]
   {
+    @extend .form_input;
+    border: 1px solid $color_placeholder;
+    margin-bottom: 0;
     width: 100%;
-    border: none;
-    outline: none;
+    height: 56px;
+  }
+}
 
-    -moz-appearance:textfield;
+.changeQtyButton
+{
+  @extend .btn_transparent;
+  border: none !important;
+  outline: none !important;
+  padding: 0;
+  width: 10px;
 
-    &::-webkit-outer-spin-button,
-    &::-webkit-inner-spin-button 
+  &:first-child
+  {
+    @include position(absolute, auto, 10px, 8px);
+    font-size: 0;
```

```suggestion
    @include fontSize(0);
```

pake mixin aja pak biar enak diliat